doc/introduction/circuits.rst

```diff
@@ -58,27 +57,19 @@ Quantum functions are a restricted subset of Python functions, adhering to the f
 constraints:
 
 * The quantum function accepts classical inputs, and consists of
-  :doc:`quantum operations <operations>` or sequences of operations called :doc:`templates`,
+  :doc:`quantum operators <operations>` or sequences of operators called :doc:`templates`,
```

Comment:
gosh the distinction between templates and operators is so arbitrary with the changes over the last year 😆

Reply:
Code-wise yes, but in research it is still very distinct :)